compiler/rustc_middle/src/ty/print/pretty.rs

```diff
@@ -760,9 +760,13 @@ pub trait PrettyPrinter<'tcx>:
                 // only affect certain debug messages (e.g. messages printed
                 // from `rustc_middle::ty` during the computation of `tcx.predicates_of`),
                 // and should have no effect on any compiler output.
```

Comment:
I don't know if this comment was accounting for UI tests manually passing `-Zverbose`... also not certain that we have to care.

Reply:
I'll mention `-Zverbose` in the comment.